manim/mobject/mobject.py

```diff
@@ -96,6 +120,37 @@ def __init__(self, color=WHITE, name=None, dim=3, target=None, z_index=0, **kwar
 
         Container.__init__(self, **kwargs)
 
+    @classmethod
+    def get_animation_override(cls, animation_class):
```

Comment:
Nonblocking, but I'd prefer this to not start with `get_`, feels a tad unpythonic. `animation_override_for` would be something I'd prefer.

Reply:
Ok, get seems to me like a good name, but I am new to python, so I'll simply trust you :D